### PR TITLE
feat(autocad): add autocad icon for openwith button

### PR DESCRIPTION
**Pull request by MonteWood**

### CODE REVIEW

**Comment by boxcla** [comment]
Hi @MonteWood, thanks for the pull request. Before we can merge it, we need you to sign our Contributor License Agreement. You can do so electronically here: http://opensource.box.com/cla

Once you have signed, just add a comment to this pull request saying, "CLA signed". Thanks!

**Comment by MonteWood** [comment]
CLA signed

--Yufeng

File: src/icons/autocad/IconAutoCAD.js [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+
+type Props = {
+    className?: string,
+    height?: number,
+    /** A text-only string describing the icon if it's not purely decorative for accessibility */
+    title?: string | React.Element<any>,
+    width?: number,
+};
+
+const IconAutoCAD = ({ className = '', height = 30, title, width = 30 }: Props) => (
```
Comment (priyajeet) [inline src/icons/autocad/IconAutoCAD.js]: I feel that this svg can likely be cleaned up. Has a lot of defs and urls.

File: src/icons/autocad/__tests__/IconAutoCAD-test.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+import IconAutoCAD from '../IconAutoCAD';
+
+describe('icons/autocad/IconAutoCAD', () => {
```
Comment (priyajeet) [inline src/icons/autocad/__tests__/IconAutoCAD-test.js]: Snapshot test helps for icons, see others for example that have snapshot tests

File: src/elements/content-open-with/IconFileMap.js [hunk 3]
```diff
@@ -4,13 +4,15 @@
...
 import IconAdobeSign from '../../icons/adobe-sign/IconAdobeSign';
+import IconAutoCAD from '../../icons/autocad/IconAutoCAD';
 import IconGoogleDocs from '../../icons/google-docs/IconGoogleDocs';
 import IconGoogleSheets from '../../icons/google-docs/IconGoogleSheets';
 import IconGoogleSlides from '../../icons/google-docs/IconGoogleSlides';
 import FileIcon from '../../icons/file-icon/FileIcon';
 
 const ICON_FILE_MAP = {
     'Adobe Sign': IconAdobeSign,
+    'AutoCAD Web App': IconAutoCAD,
```
Comment (priyajeet) [inline src/elements/content-open-with/IconFileMap.js]: Is that the actual branding for AutoCAD? Googling shows `web app` in lower case.

File: src/icons/autocad/IconAutoCAD.js (hunk 1 above)
Comment (MonteWood) [reply src/icons/autocad/IconAutoCAD.js]: sure, i will check with Zach on that, this is the raw data we are given right now

File: src/icons/autocad/__tests__/IconAutoCAD-test.js (hunk 2 above)
Comment (MonteWood) [reply src/icons/autocad/__tests__/IconAutoCAD-test.js]: will do

File: src/elements/content-open-with/IconFileMap.js (hunk 3 above)
Comment (MonteWood) [reply src/elements/content-open-with/IconFileMap.js]: the branding is actually in EUA when we add menu items, but yeah you are right the wording shall be in low case for consistency, will change it

Comment (jeremypress) [reply src/elements/content-open-with/IconFileMap.js]: @MonteWood this change is not needed since Auto Cad isn't going into the Open With Element. It needs to be referenced by the internal implementation only.

**Comment by MonteWood** [comment]
Hey, please help review this one, thanks!

File: src/icons/autocad/IconAutoCAD.js (hunk 1 above)
Comment (jstoffan) [reply src/icons/autocad/IconAutoCAD.js]: You may be able to run it through an SVG optimizer without affecting the image.

File: src/features/list-view/styles/MetadataListViewMessage.scss [hunk 4]
```diff
@@ -9,8 +9,8 @@
     }
 
     .message-header-text {
-        font-weight: bold;
         font-size: 10px;
+        font-weight: bold;
```
Comment (jstoffan) [inline src/features/list-view/styles/MetadataListViewMessage.scss]: Unrelated change?

Comment (MonteWood) [reply src/features/list-view/styles/MetadataListViewMessage.scss]: this is from linting fix, not my change, but probably we should check it in

File: src/icons/file/IconFileDwg.js [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+type Props = {
+    className?: string,
+    height?: number,
+    /** A text-only string describing the icon if it's not purely decorative for accessibility */
+    title?: string | React.Element<any>,
+    width?: number,
+};
+
+const IconFileDwg = ({ className = '', height = 32, title, width = 32 }: Props) => (
+    <AccessibleSVG
```
Comment (jstoffan) [inline src/icons/file/IconFileDwg.js]: FYI @tjgupta
